bench/wtperf/wtperf.c

```diff
@@ -1668,7 +1668,7 @@ execute_workload(CONFIG *cfg)
 	for (threads = cfg->workers, i = 0,
 	    workp = cfg->workload; i < cfg->workload_cnt; ++i, ++workp) {
 		lprintf(cfg, 0, 1,
-		    "Starting workload #%d: %" PRId64 " threads, inserts=%"
+		    "Starting workload #%u: %" PRId64 " threads, inserts=%"
```

Comment:
Should the %d to %u really be to % PRIu32?  That would be more consistent and there is no need for `i` to be `u_int` in that function.  It certainly could be uint32_t.